charts/kube-prometheus-stack/templates/alertmanager/networkpolicy.yaml

```diff
@@ -0,0 +1,79 @@
+{{- if and .Values.alertmanager.enabled .Values.alertmanager.networkPolicy.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "kube-prometheus-stack.fullname" . }}-alertmanager
+  namespace: {{ template "kube-prometheus-stack.namespace" . }}
+  labels:
+    app: {{ template "kube-prometheus-stack.name" . }}-alertmanager
+    {{- include "kube-prometheus-stack.labels" . | nindent 4 }}
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: alertmanager
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
```

Comment:
loki can also sent alerts to alertmanager.

Reply:
Added support through monitoringRules.loki: true